### PR TITLE
Text Shouldn't Wrap Around Icon in the Page Message Components

### PR DESCRIPTION
**Pull request by emilybrick**
Context in this card: 
https://buzzfeed.atlassian.net/browse/SOLID-401

- Added: float left to the icon svgs, and added a margin-left on the message text

### CODE REVIEW

File: _lib/solid-components/_messaging.scss [hunk 1]
```diff
@@ -23,11 +23,11 @@
 }
 
 .page-message__icon{
-  vertical-align: middle;
   height: $space-2;
   border-radius: 50%;
-  margin-bottom: .15rem;
   margin-right: $space-1;
+  position: relative;
+  top: .1875rem; //so the icon aligns with the first line of text
```
Comment (JackWReid) [inline _lib/solid-components/_messaging.scss]: ✨ magic number ✨  can't we just do alignment with flexbox like you said in the jira ticket?

Comment (emilybrick) [inline _lib/solid-components/_messaging.scss]: Sure - I still don't know how that would help aligning the icon with the top line of text:
screenshot 2017-07-10 15 05 33

Page messaging text can wrap infinitely, or be on a single line

Comment (emilybrick) [inline _lib/solid-components/_messaging.scss]: In lieu of something like this (vertically centered icon regardless of the lines of text), which we could achieve with flex  
screenshot 2017-07-11 08 28 49

Comment (JackWReid) [inline _lib/solid-components/_messaging.scss]: the last one 👍

**Comment by emilybrick** [comment]
@JackWReid fixed up with flexbox

**Pull request merged**